pkg/cleaner/cleaner.go

```diff
@@ -24,6 +24,9 @@ import (
 	"context"
 	"github.com/openebs/node-disk-manager/pkg/apis/openebs/v1alpha1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openebs/node-disk-manager/cmd/ndm_daemonset/controller"
```

Comment:
Usually, this shouldn't import i.e inside pkg structure cmd/ shouldn't be imported.

Reply:
Observed this from `pkg/cleanerjobcontroller.go`, let me try to get around it.